classes/system/heka/log_collector/single.yml

```diff
@@ -1,20 +1,7 @@
-#classes:
-#- service.heka.log_collector.single
 applications:
 - heka
 parameters:
-  ceilometer:
-    _support:
-      heka:
-        enabled: false
-  elasticsearch:
-    _support:
-      heka:
-        enabled: false
   heka:
-    _support:
-      heka:
-        enabled: true
```

Comment:
This actually broke `log_collector`, `metric_collector` and `aggregator` because the "heka" does not longer support "heka" with this. I created https://github.com/Mirantis/mk-lab-salt-model/pull/53 to fix this.